### PR TITLE
Documentation for the URL script output

### PR DESCRIPTION
**Pull request by emilroz**
This is the companion PR for https://github.com/ome/omero-web/pull/233

### CODE REVIEW

**Review by manics — APPROVED** [review]

File: omero/developers/scripts/style-guide.rst [hunk 1]
```diff
@@ -131,6 +131,20 @@ Script outputs
...
+-  Scripts that generate a URL link should return the omero.rtypes.rmap,
+   with the following keys: "type": "URL", "href": "URL address to open",
+   "title": "Help message". The client will give users the option
+   of opening the URL in a new browser window/tab. To use this feature 
+   the URL omero.types.rmap should use the key: 'URL' in the output map.
+
+   ::
+
+           url = omero.rtypes.rmap()
+           url.put("type", omero.rtypes.rstring("URL"))
```
Comment (will-moore) [inline omero/developers/scripts/style-guide.rst]: I'm not sure why, but it seems you need `url.put("id", rstring("-1"))` as in your example script.

File: omero/developers/scripts/style-guide.rst [hunk 2]
```diff
@@ -131,6 +131,20 @@ Script outputs
...
+           url.put("href", omero.rtypes.rstring("https://www.openmicroscopy.org))
```
Comment (will-moore) [inline omero/developers/scripts/style-guide.rst]: Missing `"`

File: omero/developers/scripts/style-guide.rst [hunk 3]
```diff
@@ -131,6 +131,20 @@ Script outputs
...
+           url.put("title", omero.rtypes.rstring("Open URL link to OME's website."))
```
Comment (will-moore) [inline omero/developers/scripts/style-guide.rst]: I wonder if it's slightly easier to use wrap:
```
        url = omero.rtypes.wrap({
            "type": "URL",
            "id": "-1",
            "href": "https://www.openmicroscopy.org",
            "title": "Open URL link to OME's website.",
        })
        client.setOutput("URL", url)
```

**Comment by emilroz** [comment]
@will-moore thanks for the comments. All should be addresses now. I also moved the URL code to a separate block so that `id` is no longer required.

**Comment by will-moore** [comment]
Looks good, thanks. 👍

**Pull request merged**